lib/Account.php

```diff
@@ -42,24 +42,119 @@ class Account extends ApiResource
     use ApiOperations\NestedResource;
     use ApiOperations\Update;
 
+    const AU_BECS_DEBIT_PAYMENTS_ACTIVE = 'active';
+    const AU_BECS_DEBIT_PAYMENTS_INACTIVE = 'inactive';
+    const AU_BECS_DEBIT_PAYMENTS_PENDING = 'pending';
+
+    const BACS_DEBIT_PAYMENTS_ACTIVE = 'active';
+    const BACS_DEBIT_PAYMENTS_INACTIVE = 'inactive';
+    const BACS_DEBIT_PAYMENTS_PENDING = 'pending';
+
     const BUSINESS_TYPE_COMPANY = 'company';
     const BUSINESS_TYPE_GOVERNMENT_ENTITY = 'government_entity';
     const BUSINESS_TYPE_INDIVIDUAL = 'individual';
     const BUSINESS_TYPE_NON_PROFIT = 'non_profit';
 
-    const CAPABILITY_CARD_PAYMENTS = 'card_payments';
-    const CAPABILITY_LEGACY_PAYMENTS = 'legacy_payments';
-    const CAPABILITY_PLATFORM_PAYMENTS = 'platform_payments';
-    const CAPABILITY_TRANSFERS = 'transfers';
-
-    const CAPABILITY_STATUS_ACTIVE = 'active';
-    const CAPABILITY_STATUS_INACTIVE = 'inactive';
-    const CAPABILITY_STATUS_PENDING = 'pending';
+    const CARD_ISSUING_ACTIVE = 'active';
+    const CARD_ISSUING_INACTIVE = 'inactive';
+    const CARD_ISSUING_PENDING = 'pending';
+
+    const CARD_PAYMENTS_ACTIVE = 'active';
+    const CARD_PAYMENTS_INACTIVE = 'inactive';
+    const CARD_PAYMENTS_PENDING = 'pending';
+
+    const CODE_INVALID_ADDRESS_CITY_STATE_POSTAL_CODE = 'invalid_address_city_state_postal_code';
```

Comment:
I really don't think we should add something like this to the codebase, but I'm obviously biased against all those constants in the first place. 

IMO those only make sense the day we start supporting real enums and API versions. Because right now, any value disappearing would be a breaking change or require an override